applications/CoSimulationApplication/python_scripts/data_transfer_operators/kratos_mapping.py

```diff
@@ -140,3 +163,301 @@ def __GetRankZeroModelPart():
             KratosMappingDataTransferOperator.__rank_zero_model_part = rank_zero_model_part
 
         return KratosMappingDataTransferOperator.__rank_zero_model_part
+
+    def __DefaultVTKSettings(self):
+        """
+        Create and return a default VTK output settings using KratosMultiphysics (KM) Parameters.
+
+        Args:
+            self: The instance of the class.
+
+        Returns:
+            KM.Parameters: A parameters object initialized with default settings for VTK output.
+        """
+        # Define default settings for VTK output
+        vtk_default_output_parameters = KM.Parameters("""{
+            "Parameters" : {
+                "model_part_name"                    : "",
+                "file_format"                        : "ascii",
```

Comment:
why ascii?

Reply:
It is ofr debugging, we like to take a direct to to the files from time to time